### DISCUSSION TITLE
Add a counter for the number of functions requests made in a given time

### DISCUSSION THREAD

**Discussion by alita-moore** [supabase/supabase#28211] [Feature Requests]
You currently have charts that show how many requests are made to the database, but it would also be nice to see how many requests are being made (in total) to all of my functions.

**Comment by GaryAustin1** [comment]
There is this chart in usage tab of settings...
![image](https://github.com/user-attachments/assets/cadcaf47-810c-4be2-a217-5095ccd8dd31)
Not sure if that is what you are looking for.

**Reply by alita-moore** [reply]
oh yeah but it's not real-time and it's also hard to find
